OOModel/src/declarations/MetaDefinition.h

```diff
@@ -41,6 +41,7 @@ class OOMODEL_API MetaDefinition : public Super<Declaration>
 {
 	COMPOSITENODE_DECLARE_STANDARD_METHODS(MetaDefinition)
 
+	ATTRIBUTE(Model::TypedList<Declaration>, arguments, setArguments)
```

Comment:
This is the formal argument of the MetaDefinition, since we do not plan on typechecking macros that means that those are just names.